Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM node:14
-WORKDIR /usr/src/app
-COPY package.json .
-COPY package-lock.json .
-COPY tsconfig.json .
+FROM node:14-alpine as builder
+RUN apk add --no-cache --virtual .build-deps python make g++
```

Comment:
virtual to create virtual package for possible removal

python, make and g++ are all required, it will not work without it.